docs/DeveloperGuide.md

```diff
@@ -133,6 +158,51 @@ Guarantees: Editing of skill or achievement is successful
   1a1. InternHunter displays an error message and informs the user of the valid input format. </br>
   Use case resumes from step 1.
 
+
+### Use case: UC14 - Get help
+
+#### MSS
+
+Precondition: User needs help with using the system.
+Guarantees: User will get directions to the user guide.
+
+MSS:
+1.  User requests for help.
+2.  InternHunter displays help message directing user to user guide. </br>
+    Use case ends.
+
+
+### Use case: UC15 - Exit 
+
+#### MSS
+
+Precondition: System is running.
+Guarantees: System will stop running.
+
+1.  User requests to exit the app.
```

Comment:
Another nitpick, maybe it's better to use InternHunter rather than the app? As the next points use InternHunter rather than the app?

Reply:
Okay will make the change!